README.md

```diff
@@ -1 +1,27 @@
-# gapminder_dashboard
\ No newline at end of file
+# Gapminder dashboard
+
+## Decription of the app
+
+Gapminder data set is a great source to know about the world. So The main objective of this app is to investigate what the young population of students know and don’t know about basic global patterns and macro-trends and educate the intended users on global development trends. This dashboard has been developed to communicate insights about historical social/economic trends and create a better understanding of using and teaching statistics.
+
+The landing page of the dashboard is designed to have five filters at the left, for selecting the target of the study, second factor of study, continent , country and the year range (1960 to 2018) .The line chart at the top is a trend-plot showing the change of the target of the study during the time over the years for the selected country, average of the continent and average of the whole world. So user would be able to compare these trend with each other. In the top right corner, there is a bubble chart which visualize the target study versus the second factor of study for all the countries.
+
+The global map is presented at the bottom of the dashboard with tooltip for each country to show the target of the study data related to the selected countries. After choosing a continent in the filter section, just the selected continent stay colourful and all other turn to grey. The level of colour shading for selected continent would be based on the value of the target of study. To the right of the global map, there is a bar plot which shows the top ten countries with highest value for target of study.
```

Comment:
The global map is presented at the bottom of the dashboard with a tooltip for each country. After choosing a continent in the filter section, the selected continent should be highlighted and the colour gradient will dictate the values of the target of study within that region. To the right of the global map, a bar plot will show the top ten countries with highest value for the selected target of study.

Reply:
Done👍🏻